misc/optlib2c

```diff
@@ -826,22 +826,22 @@ EOF
     }
     if (@{$opts->{'kinddefs'}}) {
 	print <<EOF;
-	def->kindTable = $opts->{'Clangdef'}KindTable;
-	def->kindCount = ARRAY_SIZE($opts->{'Clangdef'}KindTable);
+	def->kindTable     = $opts->{'Clangdef'}KindTable;
+	def->kindCount     = ARRAY_SIZE($opts->{'Clangdef'}KindTable);
```

Comment:
These changes about whitespace have impacts on ALL generated code under optlib/ directory.
Before adding Elixir related change, you should make a pull request for this change. I will merge it first.
The pull request should include all changes under optlib/*.c, too.

Reply:
Pull request done. There wasn't any other change under optlib/*.c, except for `optlib/elixir.c` but I think that change remains here (please correct me if I'm wrong).

How do I fix the fact that pull request #2026 and commit ffd1e98e contain the same changes? Sorry for the mess!